src/data/reformDefinitionCode.js

```diff
@@ -211,13 +211,21 @@ export function getImplementationCode(type, region, timePeriod) {
     "",
     "",
     `baseline = Microsimulation(${
-      isCountryUS ? "reform=baseline_reform" : ""
+      isCountryUS
+        ? region === "enhanced_us"
+          ? `reform=baseline_reform, dataset="enhanced_cps_${timePeriod}"`
+          : `reform=baseline_reform`
+        : ""
     })`,
-    "reformed = Microsimulation(reform=reform)",
-    `baseline_person = baseline.calc("household_net_income",`,
-    `    period=${timePeriod || defaultYear}, map_to="person")`,
-    `reformed_person = reformed.calc("household_net_income",`,
-    `    period=${timePeriod || defaultYear}, map_to="person")`,
+    `reformed = Microsimulation(${
+      region === "enhanced_us"
+        ? `reform=reform, dataset="enhanced_cps_${timePeriod}"`
```

Comment:
```suggestion
        ? `reform=reform, dataset="enhanced_cps_2022"`
```

While your assumption that this should change by year makes a lot of sense, I believe it is actually meant to map to 2022 specifically. If you try a different year in your version of the code and run the output in Google Colab, for example, you receive the following error:

```
ValueError: Dataset enhanced_cps_2023 not found. Available datasets: ['cps_2019', 'cps_2020', 'cps_2021', 'cps_2022', 'enhanced_cps_2022', 'calibrated_puf_extended_cps_2022', 'poverty_tracker', 'puf_2022', 'puf_2015']
```